ingestion_server/ingestion_server/cleanup.py

```diff
@@ -0,0 +1,208 @@
+import logging as log
+import time
+import multiprocessing
+from psycopg2.extras import DictCursor, Json
+from ingestion_server.indexer import DB_BUFFER_SIZE, database_connect
+from urllib.parse import urlparse
+
+
+# Filter out automatically generated tags that aren't of any use to us.
+# Note this list is case insensitive.
+TAG_BLACKLIST = {
```

Comment:
we may want to add license-related tags to the blacklist, see https://github.com/creativecommons/cccatalog-frontend/issues/101